owl/repl.scm

```diff
@@ -41,7 +41,7 @@
       (define (fail reason) (tuple 'fail reason))
 
       (define (name->func name)
-         (some
+         (any?
```

Comment:
The name `any?` causes some confusion. ? suffix implies the return value is a boolean, but we want the first non-false value here instead. Maybe these should use `first` in the first place, or maybe `any?` should instead be e.g. `first-value`?

Reply:
Indeed. But that's an issue it shares with `has?`. Maybe it's time to make a few steps towards the standards. For `some` we could use `any` (https://srfi.schemers.org/srfi-1/srfi-1.html#any). And instead of `has?` and `getq` we would use `memq` and `assq` everywhere, although it seems that you don't like their argument order (-;